pyproject.toml

```diff
@@ -1,3 +1,22 @@
 [build-system]
 requires = ["setuptools>=42"]
-build-backend = "setuptools.build_meta"
\ No newline at end of file
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "clmbr-altk"
+version = "0.0.1a"
+authors = [
+  { name="Shane Steinert-Threlkeld", email="shanest@uw.edu" },
+]
+description = "ALTK is a software library that aims to support efficient communication analyses of natural language. This is a line of research that aims to explain why natural languages have the structure that they do in terms competing pressures to minimize cognitive complexity and maximize communicative accuracy."
```

Comment:
Do we need all of this info in _both_ setup.py and pyproject.toml?  Can it be streamlined?  (I'm not an expert on package deployment, so genuine questions here)

Reply:
AFAIK, I think this is required in pyproject.toml, though I'll try and remove it to determine whether the build process succeeds without it. Perhaps we could add some kind of build action to generate that part of pyproject.toml?

Reply:
Different build tools make use of `setup.py` vs `pyproject.toml`, but I just learned that Python's default `setuptools` will now also read `pyproject.toml`, so we should just use that.  Everything that's currently specified in `setup.py` should be copied/moved to this file, and then we can make a very simple `setup.py` that's just 

```
from setuptools import setup

if __name__ == '__main__':
    setup()
```